lib/dal/tests/integration_test/resource_metadata.rs

```diff
@@ -27,65 +31,26 @@ use si_events::{
 };
 
 #[test]
-async fn list(ctx: &mut DalContext, nw: &WorkspaceSignup) {
-    let source_component =
-        create_component_for_default_schema_name_in_default_view(ctx, "dummy-secret", "source")
-            .await
-            .expect("could not create component");
-    let source_schema_variant_id = Component::schema_variant_id(ctx, source_component.id())
-        .await
-        .expect("could not get schema variant id for component");
-    let destination_component =
-        create_component_for_default_schema_name_in_default_view(ctx, "fallout", "destination")
-            .await
-            .expect("could not create component");
-    let destination_schema_variant_id =
-        Component::schema_variant_id(ctx, destination_component.id())
-            .await
-            .expect("could not get schema variant id for component");
-    ChangeSetTestHelpers::commit_and_update_snapshot_to_visibility(ctx)
-        .await
-        .expect("could not commit and update snapshot to visibility");
+async fn list(ctx: &mut DalContext, nw: &WorkspaceSignup) -> Result<()> {
```

Comment:
We'd still like to be able to list components! I rewrote this using modern stuff before I realized connections are really only there for secrets.